### PR TITLE
[WIP] some comments

### PR DESCRIPTION
**Pull request by ctb**

### CODE REVIEW

**Comment by ctb** [comment]
@taylorreiter done with an editing pass!

File: content/03.results.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ Limiting genomes to those that could hold at least 1% of the normalized variable
...
 
 ![
-**Phylogenetic tree of 360 bacterial species that were predictive of IBD subtype in all models.** Tree was built from the GTDB rs202 tree with all tips except those represented by the 360 genomes removed. Tree tips are labelled by genomes that anchored at least 1% of the normalized variable importance. The inner ring annotates the rank of the genomes, with the genome holding the most normalized variable importance across models ranked as 1. The outer ring is the species name within the GTDB database.
+**Phylogenetic tree of 360 bacterial species that were predictive of IBD subtype in all models.** Tree was built from the GTDB rs202 tree with all tips except those represented by the 360 genomes removed. Tree tips are labelled by genomes that anchored at least 1% of the normalized variable importance. The inner ring annotates the rank of the genomes, with the genome holding the most normalized variable importance across models ranked as 1. The outer ring is the species name within the GTDB database.  CTB: why this figure; what is the point?
```
Comment (taylorreiter) [inline content/03.results.md:96]: shrug. I feel like this is the eye candy biologists want. any suggestions for what else would go here, vs removal?

I do value having an overview of all of the genomic diversity picked up on by the models, and this was the only summary I could think of that could fit all 360 genomes.

**Review by taylorreiter — COMMENTED** [review]
Thank you!!

File: content/03.results.md (hunk 1 above)
Comment (ctb) [reply content/03.results.md:96]: hmm, brainstorming - what about a taxonomic summary instead? something tree something something?

File: content/03.results.md [hunk 2]
```diff
@@ -103,7 +103,7 @@ We hypothesized that these k-mers represented strain variable sequences not in r
 To test this hypothesis, we performed genome queries on assembly graphs of each metagenome using the 54 candidate genomes that discriminated IBD subtype (**Figure @fig:overview**).
 Assembly graph genome queries recover sequences in a metagenome that match the query, as well as those that are nearby in the assembly graph, making queries akin to but more sensitive than read mapping against reference genomes (**Figure @fig:overview**) [@doi:10.1186/s13059-020-02066-4].
 <!-- is more sensitive the right thing to say here? Should it be more general? -->
-The resulting genome query neighborhood represents a species-level umbrella that contains all sequence variation contained in the metagenome for a query.
+The resulting genome query neighborhood represents a species-level umbrella that contains all sequence variation contained in the metagenome for a query. (CTB: should we say "all" here? How do we know? This sentence seems fishy.)
```
[the remark this answers is not in the document]
Comment (ctb) [reply content/03.results.md:106]:
```suggestion
The resulting genome query neighborhood represents a species-level umbrella that contains sequence variation from the metagenome associated with a query.
```
how 'bout this?

[the remark this answers is not in the document]
Comment (taylorreiter) [reply content/03.results.md:106]: yup that's great, thank you!!

File: content/03.results.md (hunk 1 above)
Comment (taylorreiter) [reply content/03.results.md:96]: option 1: but colored by percent of variable important (maybe...that might hard. but I could try!). Pros -- shows all species. Would need to figure out how to mark species that were included in the final 54

image

option 2: summarize to a taxonomic level. Use a scatter plot to viz the % of variable importance held by those organisms. Use the size of the dot to communicate the number of species per genus. color the dot by the fraction of the final 54 that are in that taxonomic level
IMG_9470

Comment (ctb) [reply content/03.results.md:96]: I kinda like #2, I'm sorry to say :).

Comment (taylorreiter) [reply content/03.results.md:96]: cool, current or option 1 will go into supplement, and option 2 will be implemented. Thanks!

merging this PR now, but will leave your comment until I replace the figure

**Pull request merged**